en/heroku/README.md

```diff
@@ -98,6 +98,12 @@ DATABASES = {
 
 db_from_env = dj_database_url.config(conn_max_age=500)
 DATABASES['default'].update(db_from_env)
+
+# at the very end if the file add:
+try:
+    from .local_settings import *
```

Comment:
Is this something that's needed on Heroku? If not, let's leave it out. (Heroku probably has it in its own documentation, doesn't it? So people who want it can look it up there.)

Reply:
I was helping my buddy learn Django and doing his first deploy to Heroku he ran into the question of how he would alter his application locally while not having to manually switch between the settings file.  In the Heroku Docs it did not seem to include the option for `local_settings.py`, here is the links I was referring to:
https://devcenter.heroku.com/articles/django-app-configuration

It is very possible I am wrong in my thinking... I was just attempting to help out =)